src/objects/value-serializer.cc

```diff
@@ -1252,9 +1273,10 @@ ValueDeserializer::ValueDeserializer(Isolate* isolate,
       end_(data.end()),
       id_map_(isolate->global_handles()->Create(
           ReadOnlyRoots(isolate_).empty_fixed_array())) {
-
-  recordreplay::Assert("[RUN-2037] ValueDeserializer::ValueDeserializer #1 %u %d",
-                       data.size(), HashBytes(&data[0], data.size()));
+  REPLAY_ASSERT(
+    "[TT-492] ValueDeserializer::ValueDeserializer A %u %d",
+    data.size(), HashBytes(&data[0], data.size())
```

Comment:
Note: We previously hashed all this, even when not recording or Asserts were disabled.